sandbox/tests/test scenes/lights/21 - non physical light shadow off.appleseed

```diff
@@ -0,0 +1,246 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- File generated by appleseed.so version 2.0.0-beta (SSE4.2). -->
+<project format_revision="27">
+    <search_paths>
+        <search_path>
+            /home/karthik/appleseed-git/appleseed/sandbox/shaders/appleseed
```

Comment:
Since the test scenes must run everywhere, they must not contain absolute paths.